tests/firedrake/external_operators/test_external_operators_adjoint.py

```diff
@@ -77,6 +77,6 @@ def J(f):
     c = Control(f)
     Jhat = ReducedFunctional(J(f), c)
 
-    f_opt = minimize(Jhat, tol=1e-6, method="BFGS")
+    f_opt = minimize(Jhat, tol=1e-4, method="BFGS")
```

Comment:
This tolerance modification reduces the runtime test by almost 50%. Most important is that the following ``assert`` is still satisfied.